src/main/java/seedu/address/MainApp.java

```diff
@@ -74,34 +81,55 @@ public void init() throws Exception {
     }
 
     /**
-     * Returns a {@code ModelManager} with the data from {@code storage}'s address book and {@code userPrefs}. <br>
-     * The data from the sample address book will be used instead if {@code storage}'s address book is not found,
-     * or an empty address book will be used instead if errors occur when reading {@code storage}'s address book.
+     * Returns a {@code ModelManager} with the data from
+     * {@code storage} (storage's addressbook, calendar and tagtree)
+     * and {@code userPrefs}. <br>
+     *
+     * If {@code storage}'s address book/calendar is not found;
+     * The data from the sample address book/calendar will be used instead.
+     *
+     * If {@code storage}'s tagtree is not found;
+     * An empty tagtree is used instead.
+     *
+     * If errors occur when reading {@code storage}'s address book/calendar/tagtree;
+     * An empty address book/calendar/tagtree will be used instead.
      */
     private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
+        // TODO: split this method up into smaller private methods
         Optional<ReadOnlyAddressBook> addressBookOptional;
-        ReadOnlyAddressBook initialData;
+        ReadOnlyAddressBook initialAddressBook;
+
+        Optional<ReadOnlyCalendar> calendarOptional;
+        ReadOnlyCalendar initialCalendar;
+
         Optional<ReadOnlyTagTree> tagTreeOptional;
         ReadOnlyTagTree initialTagTree;
         try {
             addressBookOptional = storage.readAddressBook();
+            calendarOptional = storage.readCalendar();
             tagTreeOptional = storage.readTagTree();
             if (!addressBookOptional.isPresent()) {
-                logger.info("Data file not found. Will be starting with a sample AddressBook");
+                logger.info("AddressBook file not found. Will be starting with a sample AddressBook");
+            }
+            if (!calendarOptional.isPresent()) {
+                logger.info("Calendar file not found. Will be starting with a sample Calendar");
             }
-            initialData = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
+            initialAddressBook = addressBookOptional.orElseGet(SampleDataUtil::getSampleAddressBook);
+            initialCalendar = calendarOptional.orElseGet(SampleDataUtil::getSampleCalendar);
             initialTagTree = tagTreeOptional.orElse(new TagTreeImpl());
         } catch (DataConversionException e) {
             logger.warning("Data file not in the correct format. Will be starting with an empty AddressBook");
```

Comment:
Perhaps the warning message could be changed to incorporate Calendar too? (Not sure if it's necessary but just a thought)

Will it be starting with both empty AddressBook and empty Calendar when DataConversionException is thrown?

Reply:
Let me split this method up so that it makes more sense. If there is a DataConversionException, the one that resulted in the error should start empty, not all of them. Right now it means that if there is one DataConversionException, everything starts out empty.